deploy/kblib/templates/_rbac.tpl

```diff
@@ -10,16 +10,16 @@ Define the service account name
 {{- end }}
 
 {{/*
-Define the role name
```

Comment:
don't replace existing role and rolebinding template, define new template functions instead

Reply:
reverted